var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -138,10 +138,14 @@ class Llvm(CMakePackage, CudaPackage):
         description="Link LLVM tools against the LLVM shared library",
     )
     variant(
-        "all_targets",
-        default=False,
-        description="Build all supported targets, default targets "
-        "<current arch>,NVPTX,AMDGPU,CppBackend",
+        "targets",
+        default="all",
```

Comment:
Do you have any idea of how changing from native-only to all-targets affects build time/exe size?

Reply:
I just checked that targets=all is 7000 tasks and targets=x86 is 6000 tasks according to ninja when leaving everything else as default. Not sure what happens if also clang and other variants are turned off

Reply:
Why changing the default to 'all' here? It's certainly possible, but for some of the non-clang tools can have requirements on the system.  Recently had an error caused by a missing arm header building lldb for example, and it increases compile time non-trivially as well.  Not saying "no" mind you, but I'd like to know the reasoning.

Reply:
Sorrry, that was not intentional. Now the default is `nvptx,amdgpu` like it was.

Reply:
and by seth's comment: the default is now `none`.